db/models/order.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const {STRING, INTEGER, DOUBLE} = require('sequelize')
+
+module.exports = db => db.define('orders', {
+  orderNumber: STRING,
+  status: STRING,
+  qty: INTEGER,
+  price: DOUBLE
+}, {
+  getterMethods: {
+    getOrdNum: () =>
+      `SS-${new Date().toISOString().slice(0, 10)}-${Math.floor(Math.random()*100)+400}`
+  },
+  instanceMethods: {
+    addItem: function(item) {
+      item.orderNumber = this.orderNumber
+      return db.Order.create(item)
```

Comment:
This is creating a "child" item for the order.  The first item will be the "parent" item where the order number is created.  We could do this with an association, but it seemed more complicated that way.